doc/acl/Everflow-test-plan.md

```diff
@@ -0,0 +1,1157 @@
+
+- [Overview](#Overview)
+  - [Scope](#Scope)
+  - [Summary of the existing everflow test plan](#Summary-of-the-existing-everflow-test-plan)
+  - [Extend the test plan to cover both ingress and egress mirroring](#Extend-the-test-plan-to-cover-both-ingress-and-egress-mirroring)
+    - [What new enhancements need to be covered?](#What-new-enhancements-need-to-be-covered)
+      - [Egress ACL table](#Egress-ACL-table)
+      - [Egress mirroring](#Egress-mirroring)
+    - [Some existing areas not covered by the existing scripts](#Some-existing-areas-not-covered-by-the-existing-scripts)
+      - [ACL rule for matching "IN_PORTS"](#ACL-rule-for-matching-IN_PORTS)
+      - [IPv6 everflow](#IPv6-everflow)
+    - [How to extend the testing](#How-to-extend-the-testing)
+  - [Test configurations](#Test-configurations)
+    - [ACL table configurations](#ACL-table-configurations)
+  - [Related **DUT** CLI commands](#Related-DUT-CLI-commands)
+    - [`sonic-cfggen`](#sonic-cfggen)
+    - [`acl-loader`](#acl-loader)
+    - [`aclshow`](#aclshow)
+    - [`config mirror_session`](#config-mirror_session)
+- [Test structure](#Test-structure)
+  - [Overall structure](#Overall-structure)
+  - [Prepare some variables for testing](#Prepare-some-variables-for-testing)
+  - [Add everflow configuration](#Add-everflow-configuration)
+    - [ACL tables](#ACL-tables)
+    - [Mirror sessions](#Mirror-sessions)
+    - [ACL rules](#ACL-rules)
+  - [Run test](#Run-test)
+    - [PTF Test](#PTF-Test)
+- [Test cases](#Test-cases)
+  - [Test case \#1 - Resolved route](#Test-case-1---Resolved-route)
+    - [Test objective](#Test-objective)
+    - [Test steps](#Test-steps)
+  - [Test case \#2 - Longer prefix route with resolved next hop](#Test-case-2---Longer-prefix-route-with-resolved-next-hop)
+    - [Test objective](#Test-objective-1)
+    - [Test steps](#Test-steps-1)
+  - [Test case \#3 - Remove longer prefix route.](#Test-case-3---Remove-longer-prefix-route)
+    - [Test objective](#Test-objective-2)
+    - [Test steps](#Test-steps-2)
+  - [Test case \#4 - Change neighbor MAC address.](#Test-case-4---Change-neighbor-MAC-address)
+    - [Test objective](#Test-objective-3)
+    - [Test steps](#Test-steps-3)
+  - [Test case \#5 - Resolved ECMP route.](#Test-case-5---Resolved-ECMP-route)
+    - [Test objective](#Test-objective-4)
+    - [Test steps](#Test-steps-4)
+  - [Test case \#7 - ECMP route change (remove next hop not used by session).](#Test-case-7---ECMP-route-change-remove-next-hop-not-used-by-session)
```

Comment:
Please add test case \#6 as well

Reply:
The original missing case 6 was added. Because I combined the similar test cases, in the last version, new ID of the original case 6 is case 4.